### PR TITLE
create necessary project files and directories

### PR DESCRIPTION
**Pull request by poema-cavalcanti**
@creativespoon please review

### CODE REVIEW

**Review by creativespoon — APPROVED** [review]
Excellent setup!

File: package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    "eslint-plugin-react": "^7.14.2",
+    "babel-core": "^6.26.3",
+    "babel-loader": "^8.0.6",
+    "babel-preset-react": "^6.24.1",
+    "jest": "^24.8.0"
+  },
+  "dependencies": {
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "webpack": "^4.35.2"
```
Comment (RamKhalsa) [inline package.json:33]: @poema-cavalcanti webpack should not be a dependency for the application; if you want to list it, then the tools that it needs to run in your webpack.config.js file should be listed as dependencies as well (e.g., loaders)

File: README.md [hunk 2]
```diff
@@ -1 +1,43 @@
-# reservation
\ No newline at end of file
+# Reservation Module
```
Comment (RamKhalsa) [inline README.md:1]: I'd recommend including your endpoint to get static assets (e.g., the users initial entry into the application, how users of your microservice will request the bundle.js, etc.)

File: package.json (hunk 1 above)
Comment (poema-cavalcanti) [reply package.json:33]: I forgot to remove those from package.json when first incorporating similar feedback to not include dependencies that will be used, but aren't yet. XD

**Pull request merged**